README.md

```diff
@@ -16,27 +17,67 @@ For installation, configuration, and details about using the API, check out our
 
 For descriptions of terminology used in APM, take a look at the [official documentation](https://docs.datadoghq.com/tracing/visualization/).
 
+
+## Version Release Lines and Maintenance
+
+| Release Line                                            | Latest Version                                                                                         | Status          |Initial Release | End of Life |
+| :--:                                                    | :--:                                                                                                   | :---:           | :---:          | :---:       |
+| [`v1`](https://github.com/DataDog/dd-trace-js/tree/v1.x) | ![npm v1](https://img.shields.io/npm/v/dd-trace/legacy-v1?color=white&label=%20&style=flat-square)     | **End of Life** | 2021-07-13     | 2022-02-25  |
+| [`v2`](https://github.com/DataDog/dd-trace-js/tree/v2.x) | ![npm v2](https://img.shields.io/npm/v/dd-trace/latest-node12?color=white&label=%20&style=flat-square) | **Maintenance** | 2022-01-28     | 2023-08-15  |
+| [`v3`](https://github.com/DataDog/dd-trace-js/tree/v3.x) | ![npm v3](https://img.shields.io/npm/v/dd-trace/latest?color=white&label=%20&style=flat-square)        | **Current**     | 2022-08-15     | Unknown     |
+
+We currently maintain two release lines, namely `v2` and `v3`.
+Features and bug fixes that are merged are released to the `v3` line and, if appropriate, also the `v2` line.
+
+For any new projects it is recommended to use the `v3` release line:
+
+```sh
+npm install dd-trace
+yarn add dd-trace
+```
+
+However, existing projects that already use the `v2` release line, or projects that need to support Node.js v12 and older, may use the `v2` release line.
+This is done by specifying the version when installing the package:
+
+```sh
+npm install dd-trace@2
+yarn add dd-trace@2
+```
+
+Any backwards-breaking functionality that is introduced into the library will result in an increase of the major version of the library and therefor a new release line.
+Such releases are kept to a minimum to reduce the pain of upgrading the library.
+When a new release line is introduced the previous release line then enters maintenance mode where it will receive updates for the next year.
```

Comment:
Might also be worth mentioning that we try to follow Node.js core LTS lifecycle where we create a new release line dropping old versions when they reach end-of-life. Rather than it being dropped _entirely_ we then support them only through the legacy/maintenance branch for the one year window.

Potentially we could have other breaking changes come up in the middle of those windows, but we generally try to avoid them.